ui/userguide-view/App.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import UserGuideHeader from './UserGuideHeader';
```

Comment:
Could you separate the first and third party dependencies with a newline here too?